litmus-portal/graphql-server/server.go

```diff
@@ -126,7 +87,7 @@ func main() {
 		logrus.Fatal(err)
 	}
 
-	go startGRPCServer(rpcPort) // start GRPC server
+	go startGRPCServer(utils.Config.LitmusAuthGrpcPort) // start GRPC server
```

Comment:
I think these are 2 different variables, they might have the same values but we should keep them separate `LitmusAuthGrpcPort` is used to connect to the auth grpc server it seems and `RPC_PORT` one starts the grpc server for graphql

Reply:
Done